src/symbols.cs

```diff
@@ -0,0 +1,215 @@
+using CoreFoundation;
+using ObjCRuntime;
+using Foundation;
+
+using System;
+
+namespace Symbols {
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolEffect : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolContentTransition : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface NSSymbolEffectOptions : NSCopying, NSSecureCoding {
+		[Static]
+		[Export ("options")]
+		NSSymbolEffectOptions CreateOptions ();
+
+		[Static]
+		[Export ("optionsWithRepeating")]
+		NSSymbolEffectOptions CreateOptionsWithRepeating ();
+
+		[Export ("optionsWithRepeating")]
+		NSSymbolEffectOptions GetOptionsWithRepeating ();
+
+		[Static]
+		[Export ("optionsWithNonRepeating")]
+		NSSymbolEffectOptions CreateOptionsWithNonRepeating ();
+
+		[Export ("optionsWithNonRepeating")]
+		NSSymbolEffectOptions GetOptionsWithNonRepeating ();
+
+		[Static]
+		[Export ("optionsWithRepeatCount:")]
+		NSSymbolEffectOptions CreateOptionsWithRepeatCount (nint count);
+
+		[Export ("optionsWithRepeatCount:")]
+		NSSymbolEffectOptions GetOptionsWithRepeatCount (nint count);
+
+		[Static]
+		[Export ("optionsWithSpeed:")]
+		NSSymbolEffectOptions CreateOptionsWithSpeed (double speed);
+
+		[Export ("optionsWithSpeed:")]
+		NSSymbolEffectOptions GetOptionsWithSpeed (double speed);
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolEffect))]
+	interface NSSymbolPulseEffect {
+		[Static]
+		[Export ("effect")]
+		NSSymbolPulseEffect CreateEffect ();
+
+		[Export ("effectWithByLayer")]
+		NSSymbolPulseEffect GetEffectWithByLayer ();
+
+		[Export ("effectWithWholeSymbol")]
+		NSSymbolPulseEffect GetEffectWithWholeSymbol ();
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolEffect))]
+	interface NSSymbolBounceEffect {
+		[Static]
+		[Export ("effect")]
+		NSSymbolBounceEffect CreateEffect ();
+
+		[Static]
+		[Export ("bounceUpEffect")]
+		NSSymbolBounceEffect CreateBounceUpEffect ();
+
+		[Static]
+		[Export ("bounceDownEffect")]
+		NSSymbolBounceEffect CreateBounceDownEffect ();
+
+		[Export ("effectWithByLayer")]
+		NSSymbolBounceEffect GetEffectWithByLayer ();
+
+		[Export ("effectWithWholeSymbol")]
+		NSSymbolBounceEffect GetEffectWithWholeSymbol ();
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolEffect))]
+	interface NSSymbolVariableColorEffect {
+		[Static]
+		[Export ("effect")]
+		NSSymbolVariableColorEffect CreateEffect ();
+
+		[Export ("effectWithIterative")]
+		NSSymbolVariableColorEffect GetEffectWithIterative ();
```

Comment:
Same, simplified and property:

```suggestion
		NSSymbolVariableColorEffect Iterative { get; }
```